src/components/composer/grid_composer/grid_composer.ts

```diff
@@ -74,6 +89,39 @@ export class GridComposer extends Component<Props, SpreadsheetChildEnv> {
     });
   }
 
+  get shouldDisplayCellReference(): boolean {
+    if (this.isCellReferenceVisible) {
+      return true;
+    }
+    const sheetId = this.env.model.getters.getActiveSheetId();
+    const zone = this.env.model.getters.getSelectedZone();
+    const rect = this.env.model.getters.getVisibleRect(zone);
+    if (
+      !deepEquals(rect, this.rect) ||
+      sheetId !== this.env.model.getters.getCurrentEditedCell().sheetId
+    ) {
+      this.isCellReferenceVisible = true;
+    }
+    return this.isCellReferenceVisible;
+  }
+
+  get cellReference(): string {
+    const { col, row, sheetId } = this.env.model.getters.getCurrentEditedCell();
+    const prefixSheet = sheetId !== this.env.model.getters.getActiveSheetId();
+    return `${prefixSheet ? this.env.model.getters.getSheetName(sheetId) + "!" : ""}${toXC(
```

Comment:
it may require quotes around the sheet name.
A bit merdique, but you can use `getComposerSheetName`
or maybe we don't care

Reply:
I intentionaly don't put the quotes because it's not like in the composer. I think it's useless :)

Reply:
Consider a weird sheet name like `sheet is nice !!`, `sheet is nice !!!A1` vs `'sheet is nice !!'!A1`, shouldn't we have a consistent approach to always show the quotes by default and only support the absence of quotes and not the other way around ?

Reply:
Whatever we decide, adding a test documenting the final decision would be nice :)
(I kinda agree with Rémi. Even If I don't really care.)